js/analyzescorer.js

```diff
@@ -9,6 +9,7 @@ var isUndefined = require( "lodash/lang/isUndefined" );
 
 var assessments = {};
 assessments.wordCount = require( "./assessments/countWords.js" );
+assessments.urlLength = require( "./assessments/urlLength.js" );
```

Comment:
Consider renaming this. We're not assessing the length of the url; we're assessing whether or not the url is too long or not. Something like `urlIsTooLong` would be a better fit.